### ISSUE TITLE
Request: provide images for other platforms/architectures

### ISSUE THREAD

**Issue by white-gecko** [mathieu-lemay/python-poetry#1]
Would it be possible for you to provide the docker base image for further architectures/platforms besides `linux/amd64`, such as `linux/386,linux/arm/v6,linux/arm/v7,linux/arm64,linux/ppc64le,linux/s390x` as well?

Actually there appears to be some issue with installing poetry on other platforms then amd64. I have documented this here: https://github.com/python-poetry/poetry/issues/5131

**Comment by elsbrock** [comment]
Where can it be obtained? Is the README up to date?

**Comment by mathieu-lemay** [comment]
@elsbrock Yes, the readme is up to date.  The images can be pulled from docker hub and are available for both amd64 and arm64 architectures
